src/main/scala/com/normation/cfclerk/xmlparsers/CfclerkXmlConstants.scala

```diff
@@ -118,5 +118,6 @@ object CfclerkXmlConstants {
   val CONSTRAINT_DEFAULT = "DEFAULT"
   val CONSTRAINT_REGEX = "REGEX"
   val CONSTRAINT_PASSWORD_HASH = "PASSWORDHASH"
+  val CONSTRAINT_USED_FIELDS = "USES"
```

Comment:
I believe it should be USED here. 
In fact, USES is a terrible keywork, but I didn't find a good one for "one-side bound meaning that the field will be made available in the bounder"